debezium-server/debezium-server-redis/src/main/java/io/debezium/server/redis/RedisStreamChangeConsumer.java

```diff
@@ -149,7 +183,7 @@ public void handleBatch(List<ChangeEvent<Object, Object>> records,
                 else {
                     Transaction transaction;
                     try {
-                        LOGGER.info("Preparing a Redis Transaction of {} records", batch.size());
+                        LOGGER.trace("Preparing a Redis Transaction of {} records", batch.size());
```

Comment:
I think this kind of information should be in INFO level, like in other place throughout Debezium Server.

Reply:
Actually this has already been changed in master. This was supposed to be in trace actually.

Reply:
You are right, it was changed in a different PR.